assets/javascripts/discourse/components/explorer-schema.hbs

```diff
@@ -4,31 +4,25 @@
     @icon="chevron-left"
     @class="no-text unhide"
   />
-{{/if}}
-
-<div class="schema">
-  <div class={{if this.hideSchema "hidden"}}>
+{{else}}
+  <div class="schema">
     <div class="schema-search inline-form full-width">
-      <TextField
-        @value={{this.filter}}
-        @onChange={{this.filterChanged}}
-        @placeholderKey="explorer.schema.filter"
-      />
+      <Input {{on "input" (action "filterChanged" value="target.value")}} />
```

Comment:
To further simplify things, a standard html element can be used here now, I think:

```suggestion
      <input {{on "input" (action "filterChanged" value="target.value")}} />
```